domain/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
@joseantoniocampillo update the java version

Reply:
done